cmd/go.mod

```diff
@@ -144,3 +144,5 @@ replace (
 	github.com/containerd/stargz-snapshotter/estargz => ../estargz
 	github.com/containerd/stargz-snapshotter/ipfs => ../ipfs
 )
+
+replace github.com/hanwen/go-fuse/v2 => github.com/ktock/go-fuse/v2 v2.1.1-0.20240611080711-f59a74eb644a
```

Comment:
Do you have a PR

Reply:
Yes, there is a PR for this: https://github.com/hanwen/go-fuse/pull/520